js/ui/windowManager.js

```diff
@@ -1379,6 +1382,67 @@ const WindowManager = new Lang.Class({
                                onOverwriteParams: [shellwm, actor]
                              });
         }
+        this._endlessCodingAddLauncher(actor);
+    },
+
+    _endlessCodingAddLauncher : function(actor) {
+        let window = actor.meta_window;
+        if (window.get_flatpak_id() !== 'org.gnome.gedit')
```

Comment:
Done, I have added a whitelist. For now the list is hard-coded, can be read in from GSettings later.